packages/validation/src/Questionary/techniquePicker.ts

```diff
@@ -0,0 +1,30 @@
+import * as Yup from 'yup';
+import { AnyObject } from 'yup/lib/types';
+
+export const techniquePickerValidationSchema = (field: any) => {
+  const config = field.config;
+
+  let schema:
+    | Yup.ArraySchema<
+        Yup.NumberSchema<number | null | undefined>,
+        AnyObject,
+        (number | undefined)[] | null | undefined
+      >
+    | Yup.NumberSchema<number | null | undefined>;
+
+  if (config.isMultipleSelect) {
+    schema = Yup.array().of(Yup.number()).nullable();
+
+    if (config.required) {
+      schema = schema.required().min(1);
+    }
```

Comment:
On this do we not set the minimum number of selections.

Reply:
We are not setting any minimum number of selection while creating technique picker question. As of now, it's supporting minimum of 1 selection for the question.

Reply:
One last question are we not supposed to include the message here or we will handle it somewhere else.